tools/HelloWorld.re

```diff
@@ -10,10 +10,15 @@ module App = {
   let on_start = () => Logs.app(m => m("Running on localhost:9999"));
   type state = {random: string};
   let initial_state = {random: Random.float(0.1) |> string_of_float};
-  let route_handler: Httpkit.Server.Common.route_handler(state) =
+  let route_handler: Httpkit.Server.Common.route_handler(option(string)) =
     (ctx, path) =>
       switch (path) {
-      | [""] => `OK("hello world #" ++ ctx.state.random)
+      /* | [""] => `OK("hello world #" ++ ctx.state.random) */
+      | [""] =>
+        switch (ctx.state) {
```

Comment:
I believe `ctx.body_string` should be available since the `ctx` type has changed to include it.

The route_handler should continue to be `route_handler(state)`.

Reply:
> I believe `ctx.body_string` should be available since the `ctx` type has changed to include it.

Well for now yes, later on I suppose it would be a `read_body: unit => option(string)`, right?

> The route_handler should continue to be `route_handler(state)`

Cool, just wanted to double check 👌